### PR TITLE
Fixes #15351: Add missing upgrade rake tasks

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@jlsherrill please review

File: hooks/post/30-upgrade.rb [hunk 1]
```diff
@@ -36,6 +36,10 @@ def elasticsearch_message
   Kafo::Helpers.log_and_say :info, message
 end
 
+def add_export_distributor
+  Kafo::Helpers.execute('foreman-rake katello:upgrades:3.0:add_export_distributor')
+end
```
Comment (ehelms) [inline hooks/post/30-upgrade.rb]: @beav note that I noticed this never got added as an upgrade rake task and included it here

**Comment by jlsherrill** [comment]
ack

File: hooks/post/30-upgrade.rb (hunk 1 above)
Comment (beav) [inline hooks/post/30-upgrade.rb]: I originally added this when I was pretty new to the team, thanks for catching this :panda_face:

**Pull request merged**